sk0-back/src/users/dtos/createUser.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { IsEmail, IsNotEmpty, Matches } from 'class-validator';
+
```

Comment:
Így könnyebben olvasható:
```suggestion
const passwordRules: IsStrongPasswordOptions = {
  minLength: 6,
  minUppercase: 1,
  minLowercase: 1,
  minNumbers: 1,
  minSymbols: 0,
};

```

Reply:
módosítva, így tényleg sokkal barátságosabb!